docs/guide/uploading-downloading-binary-content.md

```diff
@@ -0,0 +1,147 @@
+# Uploading & Downloading Binary Content | AngularFire Guide
```

Comment:
A link to this page needs to get added to https://github.com/firebase/angularfire/blob/master/docs/guide/README.md. I think putting it as #4 (in between the Database and Auth sections) makes the most sense. Also, we need to make sure to update the link at the end of guide section #3 now links to this guide and, at the end of this guide, link to guide section #5 (which will now be the Auth section).